hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2017,6 +2019,49 @@ func (r *HostedClusterReconciler) reconcileControlPlaneOperator(ctx context.Cont
 		return fmt.Errorf("failed to reconcile controlplane operator rolebinding: %w", err)
 	}
 
+	// Reconcile PKI operator service account
+	controlPlanePKIOperatorServiceAccount := controlplaneoperator.PKIOperatorServiceAccount(controlPlaneNamespace.Name)
+	_, err = createOrUpdate(ctx, r.Client, controlPlanePKIOperatorServiceAccount, NoopReconcile)
+	if err != nil {
+		return fmt.Errorf("failed to reconcile controlplane PKI operator service account: %w", err)
+	}
+
+	// Reconcile PKI operator role
+	controlPlanePKIOperatorRole := controlplaneoperator.PKIOperatorRole(controlPlaneNamespace.Name)
+	_, err = createOrUpdate(ctx, r.Client, controlPlanePKIOperatorRole, func() error {
+		return reconcileControlPlanePKIOperatorRole(controlPlanePKIOperatorRole)
+	})
+	if err != nil {
+		return fmt.Errorf("failed to reconcile controlplane PKI operator role: %w", err)
+	}
+
+	// Reconcile PKI operator role binding
+	controlPlanePKIOperatorRoleBinding := controlplaneoperator.PKIOperatorRoleBinding(controlPlaneNamespace.Name)
+	_, err = createOrUpdate(ctx, r.Client, controlPlanePKIOperatorRoleBinding, func() error {
+		return reconcileControlPlanePKIOperatorRoleBinding(controlPlanePKIOperatorRoleBinding, controlPlanePKIOperatorRole, controlPlanePKIOperatorServiceAccount)
+	})
+	if err != nil {
+		return fmt.Errorf("failed to reconcile controlplane PKI operator rolebinding: %w", err)
+	}
+
+	// Reconcile PKI operator cluster role
+	controlPlanePKIOperatorClusterRole := controlplaneoperator.PKIOperatorClusterRole(controlPlaneNamespace.Name)
```

Comment:
In general, we don't bind any cluster roles to service accounts in the control plane namespace. If we need something from the infrastructure CR, the hypershift operator can get it and pass it either in the HCP or in args/env var to the PKI operator deployment.

Reply:
The `controllercmd` library uses a WATCH on this to determine how it should be doing leader election - in this case I don't think we can statically pass it in.